src/main.js

```diff
@@ -108,6 +108,8 @@ var savePosterButton = document.querySelector('.save-poster');
 savePosterButton.addEventListener('click', createRandomImage);
 var showSavedButton = document.querySelector('.show-saved');
 showSavedButton.addEventListener('click', createRandomImage);
+var makePosterButton = document.querySelector('.show-form');
```

Comment:
I felt comfortable being the navigator & felt as if I understood the concepts we were using.